javascript/javascript1/week1/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <title>Apply JavaScript example</title>
+    <script src="age-ify.js"></script>
+    <script src="dog-age.js"></script>
+    <script src="housey-pricey.js"></script>
+    <script src="Ez-namey.js"></script>
```

Comment:
This naming sticks out with the capitalized E. Consider changing it for consistency.